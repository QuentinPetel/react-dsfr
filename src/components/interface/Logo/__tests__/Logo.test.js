import { render, screen } from '@testing-library/react';
import Logo from '..';

describe('<Logo />', () => {
  it('should render Logo properly', () => {
    render(
      <Logo data-testid="logo">
        Ministère de l&apos;enseignement supérieur et de la recherche
      </Logo>,
    );
    const footer = screen.getByText(/Ministère/);
    expect(footer).toMatchSnapshot();
  });
});
