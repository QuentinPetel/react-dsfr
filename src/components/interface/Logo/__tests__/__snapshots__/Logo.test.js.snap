// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render Logo properly 1`] = `
<p
  class="fr-logo"
>
  Ministère de
  <br />
  l'enseignement
  <br />
  supérieur et de
  <br />
  la recherche
</p>
`;
